pkg/diagnostics/v2/client.go

```diff
@@ -109,6 +114,35 @@ func (c *Client) Create() (string, error) {
 	}
 }
 
+// Get gets a single bundle from the cluster
+func (c *Client) Get(id string) (*Bundle, error) {
+	req, err := c.http.NewRequest("GET", fmt.Sprintf("%s/%s", baseURL, id), nil)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := c.http.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	switch resp.StatusCode {
+	case http.StatusOK:
+		var bundle Bundle
+		err = json.NewDecoder(resp.Body).Decode(&bundle)
+		if err != nil {
+			return nil, err
+		}
+		return &bundle, nil
+	case http.StatusNotFound:
+		return nil, fmt.Errorf("no bundle %s found", id)
+	case http.StatusNotModified:
+		return nil, fmt.Errorf("bundle %s has already been deleted", id)
```

Comment:
Not directly related to this PR, but is it the right status code for this? Maybe 410 Gone would be more suited.